embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/arch/destination/LogWriterImpl.kt

```diff
@@ -36,7 +38,9 @@ class LogWriterImpl(
 
         var sessionState: String? = null
         sessionIdTracker.getActiveSession()?.let { session ->
-            builder.setAttribute(SessionIncubatingAttributes.SESSION_ID, session.id)
+            if (addSessionId) {
```

Comment:
When creating a native crash log, we don't want to attach the current sessionId as the ID, so give us a means to opt out of that.